src/WebUI/Startup.cs

```diff
@@ -52,6 +52,9 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddMvc();
 
+            //Add DAL
+            services.AddScoped<IDAL, DAL>();
```

Comment:
@emohat1r Yes, it is.
[https://codewala.net/2015/04/30/your-dependency-injection-ready-asp-net-asp-net-5/](Here ) is some more info.
P.S. it seems like ninject is not needed